registration-processor/core-processor/registration-processor-demo-dedupe-stage/src/main/java/io/mosip/registration/processor/stages/demodedupe/DemodedupeProcessor.java

```diff
@@ -285,7 +285,6 @@ public MessageDTO process(MessageDTO object, String stageName) {
 			regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), description.getCode(), registrationId,
 					description.getMessage() + ExceptionUtils.getStackTrace(e));
 			object.setInternalError(Boolean.TRUE);
```

Comment:
1. i have added below line at the end of try block
if (isTransactionSuccessful) {
				object.setIsValid(Boolean.TRUE);
			}
2. we are setting errorFlags only in case of internalError is true. that is why i have added.